assets/pages/paper-subscription-landing/components/hero/joyOfPrint.scss

```diff
@@ -80,55 +80,55 @@
 }
 
 .sale-joy-of-print-badge {
-  left: -3rem;
-  top: -0.4rem;
+  left: -4.8rem;
+  top: -0.1rem;
   position: absolute;
   flex-direction: column;
   display: flex;
   align-items: center;
   justify-content: center;
-  background: #E05E00;
+  background: #ab8b62;
   color: gu-colour(garnett-neutral-5);
   text-align: center;
   border-radius: 50%;
   line-height: 1;
   font-family: $gu-headline;
-  font-size: 14px;
```

Comment:
i'm not sure if px get converted to `rem` in our tools so for font sizes i normally always go with ems

Reply:
Good pickup :) Updated!